packages/components/src/forms/Input/index.tsx

```
@@ -50,14 +58,9 @@
   addOns?: IInputAddOnProps[];
   allowClear?: boolean; // add clear button when controlled value is not empty
   containerProps?: IGroupProps;
-  // not support on Native
-  // https://github.com/facebook/react-native/pull/45425
-  // About to add to React-Native.
-  //
-  // https://github.com/Expensify/App/pull/47203/files#diff-9bdb475c2552cf81e4b3cdf2496ef5f779fd501613ac89c1252538b008722abc
-  onPaste?: () => void;
+  onPaste?: (event: IPasteEventParams) => void;
   onChangeText?: ((text: string) => string | void) | undefined;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)